api/supabase/functions/timeline/index.ts

```diff
@@ -0,0 +1,45 @@
+import "https://esm.sh/@supabase/functions-js/src/edge-runtime.d.ts"
+import { createClient } from 'https://esm.sh/@supabase/supabase-js@2'
+
+Deno.serve(async (req) => {
+  const supaURL: string | undefined = Deno.env.get("SUPABASE_URL")
+  const supaAnonKey: string | undefined = Deno.env.get("SUPABASE_ANON_KEY")
+
+  let supabase: any
+  if (supaURL !== undefined && supaAnonKey !== undefined) {
+    supabase = createClient(supaURL, supaAnonKey)
+  }
```

Comment:
[imo]
`supaURL`もしくは`supaAnonKey`がundefinedの場合`supabase`に何も格納されないまま後続処理走ってしまうので、undefinedの場合に例外をthrowするようにするのはいかがでしょう。
そうすると`any`型使わずに済むかもしれないです。
```suggestion
  if (supaURL === undefined || supaAnonKey === undefined) {
    throw new Error('環境変数を設定してください。');
  }

  const supabase = createClient(supaURL, supaAnonKey)
```

Reply:
any型使わずに修正できました。
ありがとうございます。